sfa_dash/static/js/report-handling.js

```diff
@@ -589,6 +591,16 @@ $(document).ready(function() {
         forecast_select.change(filterObservations);
         forecast_select.change(filterAggregates);
         forecast_select.change(filterReferenceForecasts);
+        observation_select.change(function(){
+            obs_id = $(this).val();
+            if(obs_id){
+                observation = searchObjects('observations', obs_id);
+                uncertainty = observation['uncertainty'];
+                $('#selected-obs-uncertainty').html(`${uncertainty}&percnt;`);
+            }else{
```

Comment:
do you care about consistence spacing in the `if` `else` statements?

Reply:
Yes. Thank you.